tests/parser/Parser.test.js

```diff
@@ -1584,6 +1585,106 @@ describe('Parser', () => {
+        expect(parsed[0].expressions).toBeInstanceOf(Array);
+        expect(parsed[0].expressions).toHaveLength(1);
+        expect(parsed[0].expressions[0]).toBeInstanceOf(Array);
+        expect(parsed[0].expressions[0]).toHaveLength(1);
+        expect(parsed[0].expressions[0][0]).toBeInstanceOf(StandardDice);
+        expect(parsed[0].expressions[0][0]).toEqual(expect.objectContaining({
+          notation: '1d6',
+          sides: 6,
+          qty: 1,
+        }));
+      });
+
+      test('can parse `{1d6+1d4}`', () => {
+        const parsed = Parser.parse('{1d6+1d4}');
+
+        expect(parsed).toBeInstanceOf(Array);
+        expect(parsed).toHaveLength(1);
+        expect(parsed[0]).toBeInstanceOf(RollGroup);
+        expect(parsed[0].notation).toEqual('{1d6+1d4}');
+        expect(parsed[0].modifiers).toBeInstanceOf(Map);
+        expect(parsed[0].modifiers.size).toEqual(0);
+        expect(parsed[0].expressions).toBeInstanceOf(Array);
+        expect(parsed[0].expressions).toHaveLength(1);
+        expect(parsed[0].expressions[0]).toBeInstanceOf(Array);
+        expect(parsed[0].expressions[0]).toHaveLength(3);
+        expect(parsed[0].expressions[0][0]).toBeInstanceOf(StandardDice);
+        expect(parsed[0].expressions[0][0]).toEqual(expect.objectContaining({
+          notation: '1d6',
+          sides: 6,
+          qty: 1,
+        }));
+        expect(parsed[0].expressions[0][1]).toEqual('+');
+        expect(parsed[0].expressions[0][2]).toBeInstanceOf(StandardDice);
+        expect(parsed[0].expressions[0][2]).toEqual(expect.objectContaining({
+          notation: '1d4',
+          sides: 4,
+          qty: 1,
+        }));
+      });
+
+      test('can parse `{1d6, 1d4}`', () => {
+        const parsed = Parser.parse('{1d6, 1d4}');
+
+        expect(parsed).toBeInstanceOf(Array);
+        expect(parsed).toHaveLength(1);
+        expect(parsed[0]).toBeInstanceOf(RollGroup);
+        expect(parsed[0].notation).toEqual('{1d6, 1d4}');
+        expect(parsed[0].modifiers).toBeInstanceOf(Map);
+        expect(parsed[0].modifiers.size).toEqual(0);
+        expect(parsed[0].expressions).toBeInstanceOf(Array);
+        expect(parsed[0].expressions).toHaveLength(2);
+        expect(parsed[0].expressions[0]).toBeInstanceOf(Array);
+        expect(parsed[0].expressions[0]).toHaveLength(1);
+        expect(parsed[0].expressions[0][0]).toBeInstanceOf(StandardDice);
+        expect(parsed[0].expressions[0][0]).toEqual(expect.objectContaining({
+          notation: '1d6',
+          sides: 6,
+          qty: 1,
+        }));
+        expect(parsed[0].expressions[1]).toBeInstanceOf(Array);
+        expect(parsed[0].expressions[1]).toHaveLength(1);
+        expect(parsed[0].expressions[1][0]).toBeInstanceOf(StandardDice);
+        expect(parsed[0].expressions[1][0]).toEqual(expect.objectContaining({
+          notation: '1d4',
+          sides: 4,
+          qty: 1,
+        }));
+      });
+
+      test('can parse `{1}`', () => {
```

Comment:
Although the parser _can_ parse this notation, I think rolling it will most likely return something like `{1}: No dice rolled`.
The dice roller will currently only return results if there is at least 1 die roll.